doc/classes/Texture2D.xml

```diff
@@ -19,6 +19,8 @@
 			<param index="2" name="modulate" type="Color" />
 			<param index="3" name="transpose" type="bool" />
 			<description>
+				Called when the entire [Texture2D] is requested to be drawn over a [CanvasItem], with the top-left offset specified in [param pos]. [param modulate] specifies a multiplier for the colors being drawn, while [param transpose] specifies whether drawing should be performed in column-major order instead of row-major order (resulting in 90-degree clockwise rotation).
```

Comment:
I'm not sure if this description of transposed drawing is correct. Please double-check this :slightly_smiling_face:

Reply:
Transposing by itself essentially switches the x and y coordinates. It's kind of like flipping the y and rotating by 90 degrees to the right.